libdebug/debugger/internal_debugger.py

```diff
@@ -188,14 +195,16 @@ def clear(self: InternalDebugger) -> None:
 
     def start_up(self: InternalDebugger) -> None:
         """Starts up the context."""
-
         # The context is linked to itself
         link_to_internal_debugger(self, self)
 
         self.start_processing_thread()
         with extend_internal_debugger(self):
             self.debugging_interface = provide_debugging_interface()
-            self.memory = MemoryView(self._peek_memory, self._poke_memory)
+            if self.fast_memory:
+                self.memory = DirectMemoryView(self._fast_read_memory, self._fast_write_memory)
```

Comment:
Because that's what we did before, I tried to follow the same pattern.
Additionally, _peek* and _fast* methods have the preceding underscore in the name, so other classes shouldn't really call them from outside, which is why I preferred this style.
I know we do this elsewhere, but do we really want to have it in more places?

Reply:
It is just for code coherence. It is not a problem to rename them to make them 'public'; the internal debugger is intended only for internal use. This is not a blocking comment, just a suggestion.

Reply:
I'd honestly prefer to leave it as it is, this way the chunked and direct memory views are generic and can be recycled for other projects or applications, not bound to having a reference to an object implementing such methods